docs/pages/guides/environment-variables.mdx

```diff
@@ -113,7 +113,7 @@ Then update your code to use `process.env.EXPO_PUBLIC_[VARNAME]`:
 
 ### From babel-plugin-transform-inline-environment-variables
 
-Using a Babel plugin to transform your environment variable references in your code is similar to how Expo environment variables work. Set your variables inside of an **.env** file and update your variable names to use the `EXPO_PUBLIC_` prefix:
+Using a Babel plugin to transform your environment variable references in your code is similar to how Expo environment variables work. Set your variables inside an **.env** file and update your variable names to use the `EXPO_PUBLIC_` prefix:
```

Comment:
```suggestion
Using a Babel plugin to transform your environment variable references in your code is similar to how Expo environment variables work. Set your variables inside a **.env** file and update your variable names to use the `EXPO_PUBLIC_` prefix:
```

(Unrelated to this PR) but let's fix the grammatical typo here. It's read as "dot env" so the correct article will be "a" instead of "an".